packages/dds/tree/src/test/feature-libraries/editable-tree/editableTree.spec.ts

```diff
@@ -389,26 +476,8 @@ describe("editable-tree", () => {
         }
     });
 
-    it("get own property descriptor", () => {
```

Comment:
I'd prefer to keep these very targeted and simple unit tests. If they fail gives you more specific errors and are easier to understand and debug than the more general test.

On a related note, these simple tests for very specific functionality should probably be first in this file: I prefer to put the more simple or specific tests first so looking at the first failing test is a good place to start when debugging failing tests.

Reply:
Agree, I have re-structured those loops.

Reply:
Moreover, it turned out that there were no tests with `getOwnPropertyDescriptor` for symbols of `FieldProxyTarget`, which were implemented wrongly. Fixed as well.